examples/charts/file/timeseries.py

```diff
@@ -0,0 +1,51 @@
+from collections import OrderedDict
+
+import pandas as pd
+
+from bokeh.charts import TimeSeries, show, output_file, defaults, vplot, hplot
+
+# read in some stock data from the Yahoo Finance API
+AAPL = pd.read_csv(
+    "http://ichart.yahoo.com/table.csv?s=AAPL&a=0&b=1&c=2000&d=0&e=1&f=2010",
+    parse_dates=['Date'])
+MSFT = pd.read_csv(
+    "http://ichart.yahoo.com/table.csv?s=MSFT&a=0&b=1&c=2000&d=0&e=1&f=2010",
+    parse_dates=['Date'])
+IBM = pd.read_csv(
+    "http://ichart.yahoo.com/table.csv?s=IBM&a=0&b=1&c=2000&d=0&e=1&f=2010",
+    parse_dates=['Date'])
+
+xyvalues = OrderedDict(
+    AAPL=AAPL['Adj Close'],
+    Date=AAPL['Date'],
+    MSFT=MSFT['Adj Close'],
+    IBM=IBM['Adj Close'],
+)
```

Comment:
It could just be a dict I believe. I think I converted other ordereddict examples to just dicts. Using it in this way won't result in the implied order anyways.